### PR TITLE
new changes

### PR DESCRIPTION
**Pull request by aditixxwayal**
Hi, Ive made some changes in the book. can you please accept?

### CODE REVIEW

File: book.txt [hunk 1]
```diff
@@ -1,2 +1,5 @@
 Hi I am AJinkya 
-psirevo weiormgocwuehrmgqigru
\ No newline at end of file
+psirevo weiormgocwuehrmgqigru
+
+Hi Im Aditi
+asfsjnfsklfnslkfn
```
Comment (AjinkyaDhainje) [inline book.txt:5]: This is good, but add your full name and some more paras

Comment (aditixxwayal) [reply book.txt:5]: Okay will do

Comment (aditixxwayal) [reply book.txt]: I've made the changes. please look

Comment (AjinkyaDhainje) [reply book.txt]: Good work Thanks ! Merging it

**Pull request merged**